src/backend/.env

```diff
@@ -0,0 +1 @@
+PORT=3001
```

Comment:
I can see you have marked this as resolved but I cant see the original comment.  But if i am repeating then sorry.  The only thing here is from a security perspective is that you could move this file into .gitIgnore to keep it out of the repo.